packages/deployment/src/main.js

```diff
@@ -473,8 +473,13 @@ show-config      display the client connection parameters
         'SWING_STORE_TRACE',
         'XSNAP_KEEP_SNAPSHOTS',
         'NODE_HEAP_SNAPSHOTS',
-      ]) {
-        if (env[envName]) {
+      ];
+      // Use a for..in loop as env objects in node have historically allowed prototype keys
```

Comment:
thanks for the comment, good to call out why we're ok with `for..in` rather than `for..of`